README.md

```diff
@@ -23,15 +23,20 @@ and
 npx cap sync
 ```
 
+### Capacitor 3 vs Capacitor 4
+
+- Version 0.1.3 will target the latest Capacitor version
```

Comment:
Should we specify that we recommend Capacitor 3 for this version?

Reply:
Sure, I'll update this